sentry_streams/Cargo.toml

```diff
@@ -16,6 +16,7 @@ rdkafka = { version = "0.37.0", features = ["cmake-build", "tracing"] }
 anyhow = "1.0.98"
 reqwest = "0.12.15"
 tokio = "1.45.0"
+log = "0.4.27"
```

Comment:
only `tracing` should be necessary -- it exposes the same API `tracing::warn` instead of `log::warn` and so on

Reply:
sounds good